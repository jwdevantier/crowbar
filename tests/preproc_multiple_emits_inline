# <<crowbar
# @component
# def helloworld(emit):
#     emit("hello, ")
#     emit("world!")
# 
# emit(helloworld())
# >>
placeholder
# <<end>>
