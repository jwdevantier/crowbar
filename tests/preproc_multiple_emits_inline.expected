# <<crowbar
# @component
# def helloworld(emit):
#     emit("hello, ")
#     emit("world!")
# 
# emit(helloworld())
# >>
hello, world!
# <<end>>
